server/index.js

```diff
@@ -3,8 +3,8 @@ const http = require('http');
 const express = require('express');
 const app = require('./app');
 const db = require('./db/index');
-const PORT = 3000;
 
+const PORT = 3000;
```

Comment:
`const PORT = process.env.PORT || 3000` will give you the option to change the port in the env (easy for deployment)

Reply:
Thanks, I'll get that tossed in.